### ISSUE TITLE
Define conditions

### ISSUE THREAD

**Issue by johannes-z** [ehrpg/qstar#11]
A character might be subject to one or more conditions, e.g. prone, unconscious, poisoned, impeded.

**Comment by n0la** [comment]
The original idea was that such conditions just increase DCs as per the game masters discretion. But yes, status conditions are necessary to, for example, denote a player as being "prone". But whether that has actual mechanics with it, I am not sure.

Previous realistic rules overdid it (imho). For example being "prone" can give a wide variety of benefits or drawbacks depending on the situation. Someone tries to shoot you from further away? Benefit. You try to shoot someone further away from a stable prone position? Benefit. You try to shoot someone further away from an unsuitable prone position? Drawback. You try to get out of the way more quickly? Drawback. Trying to "dodge" melee hits? Drawback.

My idea would be: allow the GM to arbitrarily give or remove dice from a player or NPC depending on the conditions involved in a specific condition. Then list a few common conditions and give reasonable examples on what that could mean in terms of rules.

**Comment by johannes-z** [comment]
Should conditions add/remove dice or modifiers, or either depending on the condition?

**Comment by n0la** [comment]
> Should conditions add/remove dice or modifiers, or either depending on the condition?

Easier: The GM just gives out higher/lower DCs in case of fixed DC tasks. In competitions the GM gives out extra die to the one favoured by the condition as he or she sees fit. Any detailed ruling would be overkill. With prone alone one could fill a complete page on what sort of benefits it might or might not give. If want that sort of verbosity I can always play GURPS.

But there is in my opinion no extra ruling required. A simple:

```
## Environmental or external factors

Environmental or external factors, such as weather, conditions (such as blind or crippled), misfortune, or overwhelming odds (such as six people attacking one at the same time) are represented by removing or adding skill die to the appropriate party. The amount of die, or whom they are given to or taken away from, is up the game master's judgement of the circumstances. In the case of a skill check involving a DC, the game master should reflect the adverse or favourable conditions within the DC itself.
```

Should fix the issue. 

The conditions would then be mere labels to remember which player character is currently afflicted by said condition. A game master can then easily see who has what, and if he or she chooses to actually take further action.

tl;dr I want to avoid overly complex rules in favour of ad-hoc GM judgement

**Comment by johannes-z** [comment]
Seems as if that might become a problem as the players will never know what a condition actually does. A GM can change his judgement any time, and multiple GMs might rule this differently. A player's character could be prone, but he has no idea if he loses a die, has a higher DC, etc. Any ideas?

**Comment by n0la** [comment]
That's true. But still preferable over a page long detailed list on what prone would and would not do in every given specific circumstance.

**Comment by n0la** [comment]
So:

* We make a page describing on what certain conditions do and not do, and give a honest suggesting on structuring the rules around them. A guide on how to come up with DCs and how to give out bonuses based on circumstances.
* But we leave the detail up the GM.
* And we tell the players not to accept bullshit from the GM.

Sound reasonable?

**Comment by johannes-z** [comment]
Agreed. Nevertheless, we should collect some ideas in this or a separate issue, before we add them to the rules.
